packages/relay/src/RegistrationManager.ts

```diff
@@ -412,12 +410,17 @@ export class RegistrationManager {
     // add worker only if not already added
     const workersAdded = await this._isWorkerValid()
     const addWorkersPending = await this.txStoreManager.isActionPendingOrRecentlyMined(ServerAction.ADD_WORKER, currentBlockNumber, this.config.recentActionAvoidRepeatDistanceBlocks)
+    let skipGasEstimationForRegisterRelay = false
```

Comment:
nitpic: you actually no longer need this boolean: just move the gasLimit initialization here...